engine/src/test/scala/cromwell/CromwellTestkitSpec.scala

```diff
@@ -169,7 +184,11 @@ object CromwellTestkitSpec {
     """.stripMargin)
 
   lazy val DefaultLocalBackendConfigEntry = BackendConfigurationEntry(
-    name = "local", className = "cromwell.engine.backend.local.LocalBackend", DefaultLocalBackendConfig)
+    name = "local",
```

Comment:
Yeah it's not really used a lot, but mostly in [initBackends](https://github.com/broadinstitute/cromwell/pull/766/files#diff-aea7de37b083e8ed03b84dd9960ea9b3R44)... maybe it can be restructured so we don't need it but I struggle to see how currently... let me give it some more thought... do you have any ideas?